CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
 # Changelog
 
-As the current codebase is a pre-alpha version with many moving parts, strict versioning did not start.
\ No newline at end of file
+*September 6, 2019*
+
+The release is an incomplete alpha version meant to be deployed on the first iteration of the public testnet.
+There are no guarantees on future API and binary compatibility at this stage.
+
+## v0.0.1
+
+### Features
+* initial genesis procedure based on one-off snapshotting the ERC20 contract state and several allocation mentioned in the original whitepaper
+* initial rewards pool
+* initial configurable linear fee scheme
+* initial basic network operations related to bonded stake management
+* sketched out obfuscated transaction format
+* transaction binary payloads use SCALE
+* transaction validation in enclaves isolated in a separate process reached via a 0MQ socket
+* threshold multi-signature support using Merkle trees of combined public keys and the Schnorr MuSig scheme
+* * (for the sample use case in the context of what was marketed as "Proof of Goods & Services Delivered", see https://github.com/crypto-com/multisig-demo ) 
```

Comment:
`https://github.com/crypto-com/multisig-demo` is private

Reply:
made public